script/lib/util.py

```diff
@@ -232,7 +232,7 @@ def add_exec_bit(filename):
   os.chmod(filename, os.stat(filename).st_mode | stat.S_IEXEC)
 
 def get_out_dir():
-  out_dir = 'Debug'
+  out_dir = 'Release'
```

Comment:
By default distribution binaries output directory is `Debug`. Since in our case, we will not build binaries in Debug mode, I decided to change the default value to `Release`